src/capplet/mate-notification-applet.c

```diff
@@ -173,16 +173,19 @@ applet_main (MatePanelApplet *applet_widget)
   textdomain (GETTEXT_PACKAGE);
 #endif /* ENABLE_NLS */
 
+#ifndef ENABLE_IN_PROCESS
   g_set_application_name (_("Do Not Disturb"));
+#endif
```

Comment:
This was used for out-of-process build. Why is it set to in-process only?

Reply:
This is
`ifndef` not `ifdef`
so as to limit it to out of process not in-process builds. Running that code in-process throws a warning about setting it more than once, since the panel presumably already has an applicatio name.

If I had messed that part up I would get warnings when running the panel from terminal

Reply:
sorry for that noise.